internal/tui/components/dialog/models.go

```diff
@@ -0,0 +1,355 @@
+package dialog
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/key"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/opencode-ai/opencode/internal/config"
+	"github.com/opencode-ai/opencode/internal/llm/models"
+	"github.com/opencode-ai/opencode/internal/tui/layout"
+	"github.com/opencode-ai/opencode/internal/tui/styles"
+	"github.com/opencode-ai/opencode/internal/tui/util"
+)
+
+const (
+	numVisibleModels = 5
```

Comment:
```suggestion
	numVisibleModels = 10
```
I think we can make this a bit higher

Reply:
gotcha, if we make it a larger amount I can see why the arrows wouldnt be necessary, currently the up and down are there just to show that the view is scrollable (as in not all models are visible)